pkg/controller/windowsmachine/nodeconfig/nodeconfig.go

```diff
@@ -220,7 +222,8 @@ func (nc *nodeConfig) waitForNodeAnnotation(annotation string) error {
 	err := wait.Poll(retry.Interval, retry.Timeout, func() (bool, error) {
 		node, err := nc.k8sclientset.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})
 		if err != nil {
-			return false, errors.Wrapf(err, "error getting node %s", nodeName)
+			log.Error(err, "getting node object associated with", nc.ID())
```

Comment:
I don't follow this error message. You need to say "error getting node object associated with machine". As just `nc.ID()` won't indicate it s a `Machine`.